backends/src/trace.rs

```diff
@@ -43,7 +43,7 @@ where
         .tracing()
         .with_exporter(
             opentelemetry_otlp::new_exporter()
-                .tonic()
+                .http()
```

Comment:
Why are we making this change?

Reply:
Making it compatible with an ALB on beta platform IIRC

Reply:
Yep, I had some trouble making gRPC work in front of the load balancer. I can try again though, I'll let you know how it goes.

Reply:
Yep can confirm, it's a PITA. We have an ALB in front of OTEL, which requires an HTTPS listener when targeting a gRPC target group. The issue is that to have an HTTPS listener, you need to have a valid ACM certificate attached to it, and to get a valid ACM certificate, you need to be able to verify it, which isn't possible for internal facing address. 

I think we might be able to use ACM PCA (Private Certificate Authority), but that's not my priority right now.